models/infra_set_pol.go

```diff
@@ -65,12 +71,18 @@ func (infraSetPol *FabricWideSettingsPolicy) ToMap() (map[string]string, error)
 	}
 	A(infraSetPolMap, "annotation", infraSetPol.Annotation)
 	A(infraSetPolMap, "disableEpDampening", infraSetPol.DisableEpDampening)
+	A(infraSetPolMap, "domainValidation", infraSetPol.DomainValidation)
 	A(infraSetPolMap, "enableMoStreaming", infraSetPol.EnableMoStreaming)
 	A(infraSetPolMap, "enableRemoteLeafDirect", infraSetPol.EnableRemoteLeafDirect)
 	A(infraSetPolMap, "enforceSubnetCheck", infraSetPol.EnforceSubnetCheck)
+	A(infraSetPolMap, "leafOpflexpAuthenticateClients", infraSetPol.LeafOpflexpAuthenticateClients)
+	A(infraSetPolMap, "leafOpflexpUseSsl", infraSetPol.LeafOpflexpUseSsl)
 	A(infraSetPolMap, "name", infraSetPol.Name)
 	A(infraSetPolMap, "opflexpAuthenticateClients", infraSetPol.OpflexpAuthenticateClients)
+	A(infraSetPolMap, "opflexpSslProtocols", infraSetPol.OpflexpSslProtocols)
 	A(infraSetPolMap, "opflexpUseSsl", infraSetPol.OpflexpUseSsl)
+    A(infraSetPolMap, "policySyncNodeBringup", infraSetPol.PolicySyncNodeBringup)
```

Comment:
should this be aligned? Please run go fmt on all files

Reply:
ok done, ran gofmt on the file, guess my editor ( me blaming the editor :P ) messed things up somehow